src/se/element.rs

```diff
@@ -368,6 +370,7 @@ impl<'w, 'k, W: Write> SerializeTupleVariant for Tuple<'w, 'k, W> {
 ///   serializer
 pub struct Struct<'w, 'k, W: Write> {
     ser: ElementSerializer<'w, 'k, W>,
+    contains_non_attribute_keys:bool,
```

Comment:
Could you add a bit of documentation, explaining how this field influence serialization (when it is `true` and when is `false`, what those values means)
```suggestion
    contains_non_attribute_keys: bool,
```